extensions/wikia/AchievementsII/Ach_setup.php

```diff
@@ -261,56 +259,56 @@ function AchTest() {
 
 	global $wgUser, $wgExternalSharedDB;
 
-	set_time_limit(5000);
+	set_time_limit( 5000 );
 
-		// test 1
-		// Expect: badges for edits, categories, pictures, at least one lucky edit and eventually welcome
+	// test 1
+	// Expect: badges for edits, categories, pictures, at least one lucky edit and eventually welcome
 
-		// 2000 edits
-		for($i = 0; $i < 500; $i++) {
+	// 2000 edits
+	for( $i = 0; $i < 500; $i++ ) {
 
-			$content = date('H:i:s');
+		$content = date( 'H:i:s' );
 
-			// 2000 category edits
-			$content .= '[[Category:cat'.$i.']]';
-			$content .= '[[Category:Moja testowa]]';
-
-			// 1000 picture uploads
-			if($i % 2 == 0) {
-				$content .= '[[Image:Wiki.png]]';
-			}
+		// 2000 category edits
+		$content .= '[[Category:cat' . $i . ']]';
+		$content .= '[[Category:Moja testowa]]';
 
-			$title = Title::newFromText('CArticle_'.$i);
-			$article = new Article($title);
-			$article->doEdit($content, 'test summary');
+		// 1000 picture uploads
+		if( $i % 2 == 0 ) {
+			$content .= '[[Image:Wiki.png]]';
 		}
 
-		$dbw = wfGetDB(DB_MASTER, array(), $wgExternalSharedDB);
-		$dbw->commit();
+		$title = Title::newFromText( 'CArticle_' . $i );
+		$article = new Article( $title );
+		$article->doEdit( $content, 'test summary' );
+	}
 
-		$dbw = wfGetDB(DB_MASTER);
-		$dbw->commit();
+	$dbw = wfGetDB( DB_MASTER, array(), $wgExternalSharedDB );
+	$dbw->commit();
 
+	$dbw = wfGetDB( DB_MASTER );
+	$dbw->commit();
 
-/*
-		// test 2
-		// Expect: badges for blogposts
 
-		// 2000 edits
-		for($i = 0; $i < 2000; $i++) {
-			$content = date('H:i:s');
+	/*
+			// test 2
+			// Expect: badges for blogposts
+
+			// 2000 edits
+			for($i = 0; $i < 2000; $i++) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)